src/v3/src/components/AutoSubmit/AutoSubmit.tsx

```diff
@@ -29,7 +29,7 @@ const AutoSubmit: UISchemaElementComponent<{ uischema: AutoSubmitElement }> = ({
       step: options.step,
     });
   // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [options.isSSOExtensionVerifyEndpointAlreadyCalled]);
```

Comment:
@shawyu-okta you had a concern on this, but maybe it's moot?

Reply:
This is OK but we should put a comment here indicating it's for the SSOE fix and that it will be removed soon